L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -290,12 +290,16 @@ namespace trklet {
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
     //Following functions are used for duplicate removal
-    //Function which gets the value corresponding to the overlap size for the overlap rinv bins in DR
+    //Function which returns the value corresponding to the overlap size for the overlap rinv bins in DR
     double overlapSize() const { return overlapSize_; }
-    //Function which gets the value corresponding to the number of tracks that are compared to all the other tracks per rinv bin
+    //Function which returns the value corresponding to the overlap size for the overlap phi bins in DR
+    double phiOverlapSize() const { return phiOverlapSize_; }
+    //Function which returns the value corresponding to the number of tracks that are compared to all the other tracks per rinv bin
     unsigned int numTracksComparedPerBin() const { return numTracksComparedPerBin_; }
-    //Grabs the bin edges you need for duplicate removal bins
+    //Returns the rinv bin edges you need for duplicate removal bins
     const std::vector<double> varRInvBins() const { return varRInvBins_; }
```

Comment:
Not sure I understand the difference in naming between this and the function below -- I guess this is historical because the rinv bins are variable-width? If this is the only option, we may as well drop the var now for simplicity and symmetry with phi.

Reply:
Same thing below for varRInvBins_

Reply:
Done.

Reply:
We still have one function called "varRInvBins" and one called "phiBins". As Tova says, it's unclear why one includes the word "var", when both seem to allow variable bin size.